src/components/organisms/ChatSidebar/ChatSidebar.scss

```diff
@@ -60,6 +60,9 @@ $header-height: 45px;
   }
 
   &__controller {
+    outline: none !important;
```

Comment:
is there any way to avoid `!important`. Maybe it's better to increase CSS specificity via `div.`?

Reply:
Why are we doing this at all??!!  I've asked to keep outlines in place.  I see that there is a focus background-color applied, but our goal here is accessibility, and this is counter to that goal.

https://www.a11yproject.com/posts/2013-01-25-never-remove-css-outlines/

Reply:
👆

Reply:
@yurii-lubynets @whoaitsaimz fixed in 5f6ea55285f3f28e51ae012db3290ac4694d5c03